src/cljs/athens/views/blocks/textarea_keydown.cljs

```diff
@@ -111,40 +111,43 @@
 
 (defn filter-slash-options
   [query]
-  (if (blank? query)
-    slash-options
-    (filterv (fn [[text]]
-               (re-find (re-pattern (str "(?i)" query)) text))
-             slash-options)))
-
+  ((try
+     (if (blank? query)
+       slash-options
+       #(filterv (fn [[text]]
+                   (re-find (re-pattern (str "(?i)" query)) text))
+                 slash-options))
+     (catch js/Error _
+       slash-options))))
 
 (defn update-query
   "Used by backspace and write-char.
   write-char appends key character. Pass empty string during backspace.
   query-start is determined by doing a greedy regex find up to head.
   Head goes up to the text caret position."
   [state head key type]
-  (let [query-fn        (case type
-                          :block db/search-in-block-content
-                          :page db/search-in-node-title
-                          :hashtag db/search-in-node-title
-                          :slash filter-slash-options)
-        regex           (case type
-                          :block #"(?s).*\(\("
-                          :page #"(?s).*\[\["
-                          :hashtag #"(?s).*#"
-                          :slash #"(?s).*/")
-        find            (re-find regex head)
-        query-start-idx (count find)
-        new-query       (str (subs head query-start-idx) key)
-        results         (query-fn new-query)]
-    (if (and (= type :slash) (empty? results))
-      (swap! state assoc :search/type nil)
-      (swap! state assoc
-             :search/index 0
-             :search/query new-query
-             :search/results results))))
-
+  (try
```

Comment:
is changing this function necessary? I don't see it creating a regex from a user-supplied string.